### ISSUE TITLE
Embed linked PDF file in generated PDF

### ISSUE THREAD

**Issue by ruud12** [splitbrain/dokuwiki-plugin-dw2pdf#338]
Would it be possible to embed linked pdf files (for example {{ :test.pdf | test file}} directly into the generated pdf by mPDF? At the moment only the link is exported, but it would be a nice feature to embed all/specified pages of that pdf directly instead of just linking.

**Comment by Klap-in** [comment]
Is this related to #51 ?

**Comment by ruud12** [comment]
No, #51 is referring to attachting media files to the export. I'm referring to this feature of mPDF: https://mpdf.github.io/reference/mpdf-functions/importpage.html where actual pages of the included pdf are imported for the export.

**Comment by Klap-in** [comment]
Ah, `ImportPage()` makes then sense indeed.

I guess not everybody who has a page full of linked pdf files, prefers to import these files in the exported pdf.
` {{ :test.pdf | test file}}`  

So I'm wondering, if I would introduce this feature, is a config setting good enough for enabling this feature? I guess that the next request is a feature to set more fine-grained whether a pdf-file should be imported or only linked. 

I don't like to introduce too much config options... So I search something that is very simple, but fine-grained enough...

**Comment by ruud12** [comment]
A config setting would be fine for me, but I agree that a more fine-grained config would be better to fit everybody's needs.

Would it be possible to embed pdfs using `{{ :test.pdf?embed | test file }}`? The 'embed' argument is not recognized by dokuwiki so the link will remain the same, and this will give a fine grained control over embedding that/any particular pdf. A config setting would be nice to enable this behaviour everywhere, but that is just my personal preference
